jason_rolle/primitive_data_challenges/nba_challenge.py

```diff
@@ -0,0 +1,94 @@
+
+
+print (f'Jamal three point shot percentage {jamal_shots_percentage}%.')
+# print jamal shot percentage
+print (f'Fred three point shot percentage {fred_shots_percentage}%.')
+#print fred vanfleet shop percentage
+print (f'James three point shot percentage {james_shots_percentage}%.')
+#print james harden shop percentage
+
+
+
+information_about_lakers = 'The Lakers went all in this offseason and swung a deal for former Pelicans forward Anthony Davis.\
+ \nThey sent a package of Brandon Ingram, Lonzo Ball, Josh Hart, and 3 first-round picks to New Orleans to land Davis.\
+ \nThose three have made good developments with the Pelicans, especially Brandon Ingram.\
+ \nBut, the deal is still a huge win for the Lakers as Lebron, Davis, and company have put together an incredible season.\
+ \nLos Angeles has ridden James and Davis, along with a supporting cast built around them, to the second-best record in the NBA.\
+ \nThe Lakers ended the season atop the Western Conference with a record of 49-14.\
+ \nThey were narrowly behind the Bucks for the best record in the league.\
+ /nDavis proved to the final piece necessary for the Lakers to rebound from missing the playoffís last year.\
+ /nLos Angeles was a dominant club on both sides of the ball and are in a position to have another successful year next season.'
+# information about the lakers using escape characters such as / for a break in line
+print (information_about_lakers.upper())
+#print paragragh is capitals
+lakers_are_best = True
+#boolean is true
+print(f'The Lakers are the best basketball team in the NBA, {lakers_are_best}.')
+#print format lakers are the best
+lakers_are_best = int(lakers_are_best)
+#integer that lakers are the best
+print(lakers_are_best)
+#print lakers are the best
+lakers_are_best = float(lakers_are_best)
+#float of lakers are the best
+print(lakers_are_best)
+#print lakers are the best
+
+jamal_shots_percentage = str(jamal_shots_percentage)
+#jamal shot percentage as a string
+print (f"jamal shot percentage as a string {jamal_shots_percentage}")
+#jamal shot percentage as a string
+fred_shots_percentage = str(fred_shots_percentage)
+print (f"fred shot percentage as a string {fred_shots_percentage}")
+#fred shot percentage as a string
+james_shots_percentage = str(james_shots_percentage)
+#james shot percentage as a string
+print (f"james shot percentage as a string {james_shots_percentage}")
+#print james harden shot percentage as a string
+
+
+jamal_shots_percentage = int(jamal_shots_percentage)
+#jamal murry shot percent as an integer
+print (f"jamal shot percentage as a integer {jamal_shots_percentage}")
+# print jamal murry shot percent as an integer
+fred_shots_percentage = int(fred_shots_percentage)
+#fred vanfleet shot percent as an integer
+print (f"fred shot percentage as a integer {fred_shots_percentage}")
+# print fred vanfleet shot percent as an integer
+
+james_shots_percentage = int(james_shots_percentage)
```

Comment:
Because when you created the percentage variables you multiplied it by 100 (i.e. int(`Jamal_murray/jamal_murray_three_point_attemps*100`)) you are going to get a whole number. The purpose of the question is that if you ran the function `i.e. int(.2424)` you would get `0`. Because your variable returns a whole number, `int(james_shots_percentage)` will be the same as `str(james_shot_percentage)`